pkg/loop/internal/pb/relayerset/relayerset.proto

```diff
@@ -135,6 +135,8 @@ service EVMRelayerSet {
   rpc RegisterLogTracking(RegisterLogTrackingRequest) returns (google.protobuf.Empty);
   rpc UnregisterLogTracking(UnregisterLogTrackingRequest) returns (google.protobuf.Empty);
   rpc GetTransactionStatus(GetTransactionStatusRequest) returns (loop.evm.GetTransactionStatusReply);
+  rpc SubmitTransaction(SubmitTransactionRequest) returns (loop.evm.SubmitTransactionReply);
```

Comment:
It might be useful to have some docs on how to add a new chain capability operation. A list of all the APIs that need to be updated would be helpful if someone lacking context needs to make some changes.